draft-ietf-privacypass-architecture.md

```diff
@@ -504,6 +504,20 @@ by Origins. Moreover, as a consequence, any tokens issued by an Issuer with a
 compromised attester may no longer be trusted by Origins, even if those tokens
 were issued to Clients interacting with an uncompromised Attester.
 
+Finally, Origins can use tokens as a signal for distinguishing between Clients
```

Comment:
This paragraph is probably an entire section.  It would read better if it were three or four paragraphs, I think

Reply:
Yeah, I considered that originally, but was overcome by laziness. The latest commit moves the text to a section alongside deployment considerations like centralization, which feels like the right place for it.